src/shims-tsx.d.ts

```diff
@@ -1,13 +0,0 @@
-import Vue, { VNode } from 'vue'
```

Comment:
This "external module" is a TS-specific module system used before ES-module. It was needed in case we use .tsx files or need jsx syntax support.

ESLint has now informed me that the elements below are a duplicate declaration. The other declaration is in node_modules/vue/types/jsd.d.ts.

Therefore, I believe this file is now obsolete in our current version of Vue. (I don't see anything different in VS Code.)

Reply:
Ref: https://www.typescriptlang.org/docs/handbook/jsx.html

Might be needed only for React.